charts/mercure/values.yaml

```diff
@@ -0,0 +1,129 @@
+# Default values for mercure.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- enable the debug mode
+debug: false
+# -- enable the development mode, including the debug UI and the demo
+dev: false
+# -- the URL representation of the transport to use
+transportUrl: bolt:///data/mercure.db
+# -- extra Mercure directives to include in the Caddyfile
+extraDirectives: ""
+
+# -- the JWT key to use for publishers, a random key will be generated if empty
+publisherJwtKey: ""
+# -- the JWT algorithm to use for publishers
+publisherJwtAlg: HS256
+
+# -- the JWT key to use for subscribers, a random key will be generated if empty
+subscriberJwtKey: ""
+# -- JWT algorithm to use for subscribers
+subscriberJwtAlg: HS256
+
+# -- license key for [the High Availability version](https://mercure.rocks/docs/hub/cluster) (not necessary is you use the FOSS version):
+license: ""
```

Comment:
I still think these should be under a `config` key to better separate them from the rest of the chart values.

Reply:
Is it a common practice? AFAIK most charts don't do that (and it looks in contractions with Helm's best practices: https://helm.sh/docs/chart_best_practices/).

Reply:
I don't see a contradiction with the best practices. I don't think the claim about the mandatory check in case of nesting is true.

I honestly can't tell whether most charts do it or not. It certainly proved useful for me. Take it with a grain of salt.